pca.go

```diff
@@ -0,0 +1,79 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix"
+	"github.com/gonum/matrix/mat64"
+)
+
+// PrincipalComponents returns the principal component direction vectors of
```

Comment:
Yep, of course, thanks.

I'm not sure how to phrase it. I think "and variance values by singular value decomposition" is misleading, because it kind of suggests the singular values themselves. Maybe "PCA returns the principal component direction vectors and variance values of the matrix a, computed using the SVD of the input"? That at least puts some space between what is being returned (the variance value) and how it's being computed.